### PR TITLE
languagetool-rust: init at 2.1.1

### PR DESCRIPTION
**Pull request by name-snrl**
###### Description of changes

https://github.com/jeertmans/languagetool-rust

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/languagetool-rust/default.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, pkg-config
+, openssl
```
Comment (azahi) [inline pkgs/tools/text/languagetool-rust/default.nix:6]:
```suggestion
, openssl
, Security
```

File: pkgs/tools/text/languagetool-rust/default.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-tgx1LcVAlBcgYAdtn4n5TiLzinmOImLoatGowUFHpUM=";
+  };
+
+  cargoHash = "sha256-8Q+Li4wLkS9/HlSdtfOFnojtUBojO3oUpNHkyOu5clA=";
+
+  buildFeatures = [ "full" ];
+
+  nativeBuildInputs = [ installShellFiles pkg-config ];
+  buildInputs = [ openssl ];
```
Comment (azahi) [inline pkgs/tools/text/languagetool-rust/default.nix:25]:
```suggestion
  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security ];
```

File: pkgs/tools/text/languagetool-rust/default.nix [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+rustPlatform.buildRustPackage rec {
+  pname = "languagetool-rust";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "jeertmans";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-tgx1LcVAlBcgYAdtn4n5TiLzinmOImLoatGowUFHpUM=";
```
Comment (azahi) [inline pkgs/tools/text/languagetool-rust/default.nix:17]:
```suggestion
    hash = "sha256-tgx1LcVAlBcgYAdtn4n5TiLzinmOImLoatGowUFHpUM=";
```

**Review by SeTSeR — APPROVED** [review]
Looks good!

**Review by azahi — CHANGES_REQUESTED** [review]

File: pkgs/tools/text/languagetool-rust/default.nix (hunk 2 above)
Comment (SeTSeR) [reply pkgs/tools/text/languagetool-rust/default.nix:25]: Is there any documentation entry on this `Security` on Darwin and why is it needed? Or could you provide some clarification?

**Review by azahi — APPROVED** [review]

**Pull request merged**